infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -236,6 +237,17 @@ public void eth1DepositChainIdMismatch(int expectedChainId, int eth1ChainId) {
         eth1ChainId);
   }
 
+  public void externalSignerStatus(final URL externalSignerUrl, boolean isReachable) {
+    if (isReachable) {
+      log.info("External signer is reachable at {}", externalSignerUrl);
```

Comment:
this is happening every 30 seconds, I can understand logging when it wasn't available and now it is back, but if it's been available I'd prefer this is a debug level... Most of the time this will equate to log noise...

Reply:
actually, it does look like this isn't every 30 seconds but on each change... if that's the case I'm happy 👍

Reply:
yes, log all failures, once after every success.